cypress/e2e/about.cy.ts

```diff
@@ -3,16 +3,13 @@ import enUS from '../../i18n/locales/en.json'
 describe('About page', () => {
     context('Desktop resolution', () => {
         before(() => {
-            cy.visit('/about')
-            // This wait time is to give the page time to load from Prod when ran in CI.
-            cy.wait(3000)
+            cy.visit('/about', { timeout: 10000 })
```

Comment:
👏 soooo happy to see a timeout instead! this is much better than artifical wait times! nice find